src/PartielsPy/Partiels.py

```diff
@@ -0,0 +1,76 @@
+"""A main class for Partiels Wrapper"""
+
+import os
+import platform
+import shutil
+import subprocess
+import semver
+
+
+class Partiels:
+    def __init__(self):
+        """Initialize Partiels executable path and version
+
+        The executable path is determined by searching the system's PATH environment variable.
+        If the executable is still not found, it checks common installation directories based on the operating system.
+        You can use the PARTIELS_PATH environment variable to set the executable path.
```

Comment:
It would be better to specify that in this case, only the  `PARTIELS_PATH` is used.